### PR TITLE
Improved unit tests to eliminate spurious failures

### PR DESCRIPTION
**Pull request by tombogle**
These tests formerly relied on clipboard access or Application.DoEvents and actual mouse positions and timing.

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1365)

### CODE REVIEW

File: SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs [hunk 1]
```diff
@@ -95,6 +96,13 @@ public void ClipboardRoundTripWorks_GetsExistingMetadata()
 		[Platform(Exclude = "Linux", Reason = "This requires a GTK message loop to run or something like this")]
 		public void ClipboardRoundTripWorks_Text()
 		{
+			if (Environment.GetEnvironmentVariable("CI") == null)
```
[the remark this answers is not in the document]
Comment (josephmyers) [reply SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs:99]: What's the reason to only mark this test? Are the others fine?

**Review by josephmyers — DISMISSED** [review]

File: SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tombogle) [reply SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs]: It wasn't the other day, but now it is. Maybe fixed by the previously merged PR?

**Review by andrew-polk — APPROVED** [review]
> Dismissed @josephmyers from a discussion.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1365)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

**Review by andrew-polk — APPROVED** [review]
> Dismissed @tombogle from a discussion.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1365)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @tombogle)

**Pull request merged**